cmd/minikube/cmd/stop.go

```diff
@@ -77,7 +77,7 @@ func runStop(cmd *cobra.Command, args []string) {
 	}
 
 	machineName := pkg_config.GetMachineName()
-	err = kubeconfig.UnsetCurrentContext(constants.KubeconfigPath, machineName)
+	err = kubeconfig.UnsetCurrentContext(machineName, constants.KubeconfigPath)
```

Comment:
This PR caused me to LOL, but not necessarily in a good way. 

There is a second bug here: `constants.KubeconfigPath` does not respect the `KUBECONFIG` environment variable. If the filename isn't provided, we do the right thing. Here is my suggested change to reduce the number of paths to sadness in our code base:

- Change *SetCurrentContext to either take a second argument, or not take a second argument. Having it support both is a recipe for disaster.
- If you prefer the 1-argument version, update `context_test.go` to set the temp path via an alternative mechanism, such as `KUBECONFIG`
- If you prefer the 2-argument version, update these the `kubeconfig.*Context()` calls to pass  `kubeconfig.PathFromEnv()`.